### ISSUE TITLE
how to train the custom models in tensorflow 2.0 please help me out

### ISSUE THREAD

**Issue by sreenupadidapu** [TannerGilbert/Tensorflow-Object-Detection-API-Train-Model#2]

**Comment by TannerGilbert** [comment]
Sorry for the late reply. Didn't notice the issue. Currently training with the Tensorflow Object Detection API isn't supported with Tensorflow 2 yet. You'll have to use Tensorflow 1.

**Comment by sreenupadidapu** [comment]
then when we use tensorflow 2.0 for Tensorflow object detection API

**Comment by TannerGilbert** [comment]
Training a object detection with Tensorflow 2 is now supported. I have updated the repository for Tensorflow 2. If someone still wants to use Tensorflow 1 he can use the [tf1 branch](https://github.com/TannerGilbert/Tensorflow-Object-Detection-API-Train-Model/tree/tf1).

**Comment by gunarms** [comment]
The thing is I have already trained a model and I believe I have exported it correctly but I am stuck with a model that I have no idea what my next step is in actually implementing the model with some code to actually use it.

From Oscar

On 31 Jul 2020, at 4:44 am, Tanner Gilbert <notifications@github.com> wrote:

﻿

Training a object detection with Tensorflow 2 is now supported. I have updated the repository for Tensorflow 2. If someone still wants to use Tensorflow 1 he can use the tf1 branch<https://github.com/TannerGilbert/Tensorflow-Object-Detection-API-Train-Model/tree/tf1>.
